modules/IRCChat.js

```diff
@@ -0,0 +1,138 @@
+var irc = require('irc');
+
+var IRCConnections = {};
+
+function IRCChat(host, server_port, name, pass, channel, WelcomeMsg, isSecure, cmd_prefix) {
```

Comment:
When you get to this many params, you're probably better off using a config object instead of worrying about function parameter order.